src/njoy21/input/PLOTR/src/readPlots.hpp

```diff
@@ -0,0 +1,6 @@
+template< typename Istream >
```

Comment:
The filename and function name are out of sync. It appears (from the syntax) that you are reading a single Plot thus the filename should be changed to be `readPlot.hpp`.